src/main/java/com/cloudant/client/internal/views/ViewQueryParameters.java

```diff
@@ -97,7 +97,7 @@
      *
      * @param parameters to copy information from
      */
-    public ViewQueryParameters(ViewQueryParameters<K, V> parameters) {
+    ViewQueryParameters(ViewQueryParameters<K, V> parameters) {
```

Comment:
There's a bunch of things you're making non-public in this stateless pagination commit. Is that changing of visibility a necessary step as part of that commit; it feels like it could be a separate commit.